README.rst

```diff
@@ -1 +1,15 @@
-Please refer to src/zope/globalrequest/README.rst.
+zope.globalrequest
+==================
+
+Introduction
+------------
+
+This package provides a global way to retrieve the currently active request object in a zope-based web framework.
```

Comment:
Maybe it's just me, but when I see "global" I have questions about thread safety.  Perhaps mention "the currently active request object of the current thread"?  Would that make the description too unwieldy?  Maybe put "(of the current thread)" in parentheses?